authentication/README.md

```diff
@@ -0,0 +1,21 @@
+# Authentication 
+## Introduction
+
+This tutorial will introduce the basics of authentication, implemented for both websites and mobile applications.  The code and ideas presented here **should only be used in a hackathon environment**.  In a production environment, most of the tatics listed here would not be secure enough, and may leave ways for determined hackers to access sensitive information.  The code here utilizes mostly Javascript and MongoDB.
```

Comment:
Can you add a quick sentence on what authentication is and when you should worry about adding it to a project and when it's not necessary? (though should be pretty obvious)

Reply:
Oops, I didn't see that I didn't finish the paragraph.  I will add everything requested soon.